tools/chpl-language-server/src/chpl-language-server.py

```diff
@@ -523,7 +530,7 @@ class FileInfo:
     ] = field(init=False)
     siblings: chapel.SiblingMap = field(init=False)
     used_modules: List[chapel.Module] = field(init=False)
-    possibly_visible_decls: List[chapel.NamedDecl] = field(init=False)
+    visible_decls: List[Tuple[str, chapel.AstNode]] = field(init=False)
```

Comment:
Is the `core-types.h` available to be used, so this could be `List[VisibleSymbol]` or `VisibleSymbols`?

Reply:
type aliases a la `using` are not transformed into (Python) types automatically. That would require hooking deeper into C++ than macros and templates. Alas, unless I define a wrapper type -- which I don't want to do -- this will be a list of tuples.

Reply:
Ok I was misremembering what we do for QualifiedType. Might be worth adding a similar type alias in __init__.py. But not that big a deal